From 1d63dfe1ff51250d0785dd32b830f1eb0984c383 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Sun, 29 May 2022 15:43:22 -0700
Subject: [PATCH 11/17] NOT-CBL: watchdog: hpe-wdt: Add MODULE_LICENSE

  ERROR: modpost: missing MODULE_LICENSE() in drivers/watchdog/gxp-wdt.o

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 drivers/watchdog/gxp-wdt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/watchdog/gxp-wdt.c b/drivers/watchdog/gxp-wdt.c
index b0b2d7a6fdde..2fd85be88278 100644
--- a/drivers/watchdog/gxp-wdt.c
+++ b/drivers/watchdog/gxp-wdt.c
@@ -172,3 +172,4 @@ module_platform_driver(gxp_wdt_driver);
 MODULE_AUTHOR("Nick Hawkins <nick.hawkins@hpe.com>");
 MODULE_AUTHOR("Jean-Marie Verdun <verdun@hpe.com>");
 MODULE_DESCRIPTION("Driver for GXP watchdog timer");
+MODULE_LICENSE("GPL");
-- 
2.36.1

